src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,68 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../../components/Loader';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { PeopleFilters } from '../../components/PeopleFilters';
+import { PeopleTable } from '../../components/PeopleTable';
+import { mappedPeople } from '../../utils/proceedPeople';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isError, setIsError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const displayPeopleTable = () => (
+    !isError && !isLoading && !!people.length
```

Comment:
```suggestion
  const shouldDisplayPeopleTable = !isError && !isLoading && !!people.length
```
Make it just a variable, no need to create a function